pkg/hash/hash.go

```diff
@@ -129,7 +127,7 @@ func (hash *Hash) Clone() *Hash {
 
 // CloneWithID returns a copy of the Hash in its current state, but also writes the ID to the new state.
 func (hash *Hash) CloneWithID(id party.ID) *Hash {
-	h2 := hash.h.Clone()
-	_, _ = h2.Write([]byte(id))
-	return &Hash{h: h2}
+	cloned := hash.Clone()
+	cloned.Write([]byte(id))
```

Comment:
Linters sometimes complain with unhandled errors, in most places I prefer to put `_,_ = ...`.

Reply:
Oh this was an oversight on my part